hadoop-examples/build.gradle

```diff
@@ -35,7 +34,7 @@ repositories {
 }
 
 dependencies {
-    compile     "io.pravega:hadoop-connectors:${connectorVersion}"
+    compile     "io.pravega:pravega-connectors-hadoop:${hadoopConnectorVersion}"
```

Comment:
After trying the changes from this PR locally, the example fails throwing exceptions like: 
```
java.lang.Exception: io.pravega.client.segment.impl.NoSuchSegmentException: Segment does not exist:myScope/myStream/2
```
I found similar exceptions when testing examples including versions of `pravega-client` older than the major changes merged in the last days (e.g., rolling transactions). Thus, to make this example work out-of-the-box, maybe we will need to configure a newer version of the hadoop connector build with libraries containing the recent major changes in pravega.

Reply:
I have created an issue for this https://github.com/pravega/hadoop-connectors/issues/36 yesterday and @yangb8 is working on it.